R/create.phylogenetic.tree.R

```diff
@@ -0,0 +1,37 @@
+create.phylogenetic.tree <- function(
+    tree,
+    filename = NULL,
+    width = 5,
+    height = 5,
+    units = 'in',
+    res = 500,
+    bg = 'white',
+    ...
+    ) {
+
+    if ('node.id' %in% colnames(tree)) {
+        rownames(tree) <- tree$node.id;
+        }
+
+    plt <- SRCGrob(
+        tree,
+        ...
+        );
+
+    supported.extensions <- c('png', 'pdf', 'tiff', 'svg');
+    if (!is.null(filename)) {
+        save.plot(
```

Comment:
Does BPG's `write.plot` function work here, or is it designed to only work with lattice plots?